Measurements/Measurements_Workbook.ipynb

```diff
@@ -0,0 +1,453 @@
+    "This workbook describes the solutions to the problems offered in the [Measurement Kata](https://notebooks.gesis.org/binder/jupyter/user/microsoft-quantumkatas-iacsymjt/notebooks/Measurements/Measurements.ipynb). Since the tasks are offered as programming problems, the explanations also cover some elements of Q# that might be non-obvious for a first-time user.\n",
+    "\n",
+    "**What you should know for this workbook?**\n",
+    "\n",
+    "You should be familiar with the following concepts before tackling the Measurements kata (and this workbook):\n",
+    "\n",
+    "1.\tBasic linear algebra\n",
+    "2.\tThe concept of qubit and multi-qubit systems\n",
+    "3.\tSingle-qubit and multi-qubit quantum gates\n",
+    "4.\tSuperposition\n",
+    "\n",
+    "To begin, first prepare this notebook for execution (if you skip the first step, you'll get \"Syntax does not match any known patterns\" error when you try to execute Q# code in the next cells; if you skip the second step, you'll get \"Invalid test name\" error):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%package Microsoft.Quantum.Katas::0.10.1911.1607"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%workspace reload"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part I. Discriminating Orthogonal States"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Task 1.1.  $|0\\rangle$ or $|1\\rangle$ ?\n",
+    "\n",
+    "**Input:** A qubit which is guaranteed to be in either the $|0\\rangle$ or the $|1\\rangle$ state.\n",
+    "**Output:**  `true` if the qubit was in the $|1\\rangle$ state, or `false` if it was in the $|0\\rangle$ state. The state of the qubit at the end of the operation does not matter."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Solution**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "operation IsQubitOne (q : Qubit) : Bool {\n",
+    "\treturn (M(q) = One);\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When a qubit is measured its superposition state collapses, meaning that the superposition state (a linear combination of basis vectors) collapses to just one of the basis vectors. Since in this task the original state of the qubit is already one of the basis vectors, we know measurement would identify that state exactly. \n",
+    "[M( )] (https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.primitive.m?view=qsharp-preview) is the measurement gate, it returns true if the qubit is in state 1 and false if the qubit is in state 0.\n",
```

Comment:
Markdown doesn't recognize this as link - that extra space breaks the connection between link text and the link itself.
```suggestion
    "[M( )](https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.primitive.m?view=qsharp-preview) is the measurement gate, it returns true if the qubit is in state 1 and false if the qubit is in state 0.\n",
```